plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorSmokeTest.java

```diff
@@ -14,6 +14,20 @@
 package io.trino.plugin.jdbc;
 
 import io.trino.testing.BaseConnectorSmokeTest;
+import io.trino.testing.TestingConnectorBehavior;
 
 public abstract class BaseJdbcConnectorSmokeTest
-        extends BaseConnectorSmokeTest {}
+        extends BaseConnectorSmokeTest
+{
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        switch (connectorBehavior) {
```

Comment:
Why no SUPPORTS_DELETE, SUPPORTS_CREATE_VIEW as in BaseJdbcConnectorTest?
Does it indicate haps in the smoke test?